### ISSUE TITLE
CLI: multi-direction/multi-speed inputs

### ISSUE THREAD

**Issue by VVionnet** [firelab/windninja#340]
The CLI supports only one input wind speed and direction whereas the GUI offers the possibility to have multiple input wind speed and direction. This possibility of the GUI is particularly interesting when using NinjaFoam since the mesh is built only once, saving some computational time for large domain. Would it be possible to add the option of multiple input wind speed and direction in the CLI? 
Thanks!

**Comment by nwagenbrenner** [comment]
@VVionnet thanks for the suggestion. I agree this would be useful, and probably not too much work to implement.

**Comment by ncubukcu** [comment]
I saw this in GUI and wasn't sure how to interpret that. Is this GUI feature only for different times? Otherwise it would be the same thing as in point initialization. Thanks

**Comment by nwagenbrenner** [comment]
Yes, the GUI allows you to do multiple domain average runs. The runs are all independent. This option would be used if you wanted to run multiple simulations with different input conditions (speed, direction, time of day, cloud cover, etc.).
